x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/service/RestGetServiceAccountAction.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.security.rest.action.service;
+
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.license.XPackLicenseState;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.xpack.core.security.action.service.GetServiceAccountAction;
+import org.elasticsearch.xpack.core.security.action.service.GetServiceAccountRequest;
+import org.elasticsearch.xpack.security.rest.action.SecurityBaseRestHandler;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.elasticsearch.rest.RestRequest.Method.GET;
+
+public class RestGetServiceAccountAction extends SecurityBaseRestHandler {
+
+    public RestGetServiceAccountAction(Settings settings, XPackLicenseState licenseState) {
+        super(settings, licenseState);
+    }
+
+    @Override
+    public List<Route> routes() {
+        return List.of(
+            new Route(GET, "/_security/service"),
+            new Route(GET, "/_security/service/{namespace}"),
+            new Route(GET, "/_security/service/{namespace}/{service}")
+        );
+    }
+
+    @Override
+    public String getName() {
+        return "xpack_security_get_service_account";
+    }
+
+    @Override
+    protected RestChannelConsumer innerPrepareRequest(RestRequest request, NodeClient client) throws IOException {
+        final String namespace = request.param("namespace");
+        final String serviceName = request.param("service");
```

Comment:
Should we support `*` here? 
I don't think we _need_ to, but I wonder if we should for consistency with other APIs.
Although, that then opens the question of comma separated lists, etc.

Reply:
I thought about it and decide to keep it simple for now. Also wildcard (`*`) is not supported for APIs like GetRole or GetUser. The main reason is likely because their names allow the literal `*` character. Neither service account namespace nor service-name allow it, so technically it can support. But I decided not to for somewhat consistency and simplicity. 
The comma separated list is more widely supported. It can also be supported here as well. But again I decided for simplicity (both `namespace` and `service` can support comma separated list, the natural relation is likely combination of the two sets, but we haven't really talked about it). 
Given we only have a single service account, I think it is viable to have this simplest form first. It is easy to expand once the need arises.

Reply:
PS: GetUser and GetRole has a minor bug when it comes to comma separated lists. Because they both allow comma to be used as the name, this means if we created an user `a,b`, we cannot get it back by itself anymore. The API will interpret `GET _security/user/a,b` as retrieve user `a` and `b` and returns an empty result.